src/Template/Bake/Controller/component.twig

```diff
@@ -0,0 +1,34 @@
+{#
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         2.0.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+#}
+<?php
+namespace {{ namespace }}\Controller\Component;
+
+use Cake\Controller\Component;
+use Cake\Controller\ComponentRegistry;
+
+/**
+ * {{ name }} component
+ */
+class {{ name }}Component extends Component
+{
+
+    /**
```

Comment:
According to psr2 there shouldnt be newlines after opening brace. Not sure if the sniffer would auto-fix them afterwards, but it should.

Reply:
Currently my aim is to generate the same output as the `.ctp` templates do. Any whitespace changes will cause the tests to fail. If the generated output is incorrect then it can be fixed for both type of templates later.